File: Chapter 1/(g)/g.java
import java.util.Scanner;

class g
{
	static public void main(String args[])
	{
		Scanner input=new Scanner(System.in);
		int sum;
		int num;
		sum=0;
		System.out.print("Enter the number (digits <5)");
		num=input.nextInt();

		for(int i=0;i<num;++i)
		{
			sum=sum+ num%10;
			num=num/10;
		}

		System.out.println("Sum of all digits is "+sum);



	}
}

File: Chapter 1/(g)/g.c
#include <stdio.h>

int main(int argc, char *args[])
{
	unsigned long num;
	int sum;
	sum = 0;
	printf("Enter The Number(< 5 digits) : ");
	scanf("%u", &num);

	for (int i = 0; i < 5; ++i)
	{
		sum = sum + (num % 10);
		num = num / 10;
	}

	printf("Sum of all digits in input number is = %d ", sum);

	return 0;
}